ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -353,4 +353,10 @@ if get_option('enable-openvino')
     install_dir: nnstreamer_libdir
   )
 
+  nnstreamer_filter_openvino_dep = declare_dependency(
+    link_with: nnstreamer_filter_openvino_so_lib,
+    compile_args : openvino_cpp_args,
+    dependencies: [nnstreamer_filter_openvino_deps],
```

Comment:
You don't need Line 358-360. The code using libnnstreamer_filter_openvino.so does not inherit the build-dependencies of libnnstreamer_filter_openvino.so.